internal/config/config.go

```diff
@@ -37,6 +37,10 @@ type PullRequest struct {
 
 	// Branch prefix to use for creating new branches.
 	BranchNamePrefix string
+
+	// If true, the CLI will automatically add/update a comment to all PRs linking other PRs in the stack.
+	// False by default, since Aviator's MergeQueue also adds a similar comment.
+	WriteStack bool
```

Comment:
I could see this being on by default if there was a way for `av` to auto-detect if `MergeQueue` is being used? Or alternatively if this is turned on then maybe it's not needed in the MergeQueue comments? (I haven't used myself yet.)